FHICORC/Views/Elements/NavigationHeader.xaml

```diff
@@ -8,23 +8,26 @@
     x:Name="HeaderGrid"
     BackgroundColor="{StaticResource NavigationHeaderBackgroundColor}">
 
-    <Grid Margin="10,10">
+    <Grid Margin="10,10"
+          ColumnSpacing="10"
+          ColumnDefinitions="Auto, *, Auto">
         <ImageButton
             x:Name="LeftButton"
+            Grid.Column="0"
             Padding="6,6,6,6"
             AutomationId="LeftButton"
             BackgroundColor="Transparent"
             HorizontalOptions="Start"
             VerticalOptions="Center" />
-
         <Label
             x:Name="CenterLabel"
+            Grid.Column="1"
             local:FontSizeLabelEffectParams.MaxFontSize="32"
             local:FontSizeLabelEffectParams.MinFontSize="12"
             AutomationId="{Binding CenterLabelText}"
             HorizontalOptions="CenterAndExpand"
             HorizontalTextAlignment="Center"
-            LineBreakMode="NoWrap"
+            LineBreakMode="TailTruncation"
```

Comment:
I think the whole header should show, either in two lines or with a smaller text size

Reply:
Updated max lines to two and fixed center of label.